README.md

```diff
@@ -1 +1,11 @@
 # discourse-gated-topics-in-category
+This theme component restricts users from accessing topics from specific categories. A prompt will be displayed to encourage logins.
```

Comment:
I would recommend rephrasing this just to note that the component only does this visually, i.e. this is NOT a security feature. Maybe we should say 
```suggestion
This theme component allows site admins to show a login prompt that covers the topic contents in specific categories as a way to encourage readers to sign up and log in.
```

Reply:
Good point!